docs/reference/extensions/ros-2-content-extensions.rst

```diff
@@ -0,0 +1,73 @@
+.. _ros-2-content-extensions:
+
+ROS 2 Content extensions
+========================
+
+The ROS 2 Content extensions comprise the main ROS 2 extensions, plus additional
+settings to enable `content sharing <https://snapcraft.io/docs/content-interface>`_.
+
+These extensions are split across ROS 2 versions and content types, and are declared
+with the format ``ros2-<version>-<metapackage>``. The available extensions are:
+
+- ``ros2-foxy-ros-core``
+- ``ros2-foxy-ros-base``
+- ``ros2-foxy-desktop``
+- ``ros2-humble-ros-core``
+- ``ros2-humble-ros-base``
+- ``ros2-humble-desktop``
+- ``ros2-jazzy-ros-core``
+- ``ros2-jazzy-ros-base``
+- ``ros2-jazzy-desktop``
```

Comment:
What about tabbing this based on the version of ROS 2 you're using? You should never use the `humble` content extensions with the main `jazzy` extension for example.

Reply:
Good idea! 9926591